src/styles/variables.less

```diff
@@ -1,4 +1,4 @@
-@card-icon-lg:                                        108px;
+@card-icon-lg:                                        72px;
```

Comment:
Since the value of @card-icon-lg is now the same as @card-icon-sm, we don't both in addition to the overrides that make use of @card-icon-lg.